tests/components/philips_js/test_diagnostics.py

```diff
@@ -0,0 +1,65 @@
+"""Test the Philips TV diagnostics platform."""
+
+from unittest.mock import AsyncMock
+
+from syrupy import SnapshotAssertion
+from syrupy.filters import props
+
+from homeassistant.core import HomeAssistant
+
+from tests.common import MockConfigEntry
+from tests.components.diagnostics import get_diagnostics_for_config_entry
+from tests.typing import ClientSessionGenerator
+
+TV_CONTEXT = {"level1": "NA", "level2": "NA", "level3": "NA", "data": "NA"}
+TV_CHANNEL_LISTS = {
```

Comment:
Could you use the typed dict values from the library for this. Less likely we break stuff on upgrade.

Reply:
https://github.com/danielperna84/ha-philipsjs/blob/master/haphilipsjs/typing.py

Reply:
Done

Reply:
Just realized, the library install dummy data that we could have used here: https://github.com/danielperna84/ha-philipsjs/tree/master/haphilipsjs/data

But im okey with this.